Jenkinsfile

```diff
@@ -21,13 +21,16 @@ def releaseTag = new com.socrata.ReleaseTag(steps, service_server)
 pipeline {
   options {
     ansiColor('xterm')
+    buildDiscarder(logRotator(numToKeepStr: '50'))
+    disableConcurrentBuilds(abortPrevious: true)
+    timeout(time: 70, unit: 'MINUTES')
```

Comment:
Deploy of soql-server is set to max out at 60 min.  The rest of the build usually takes 6 min at most.